### DISCUSSION TITLE
Nextjs build misses files for sublocales

### DISCUSSION THREAD

**Discussion by amurashev** [vercel/community#1714] [Help] [answered]
### Summary

After some changes (probably https://github.com/vercel/vercel/pull/9582 or any changes after March 2nd) nextjs build started to miss files for sublocales (masked locales?). 

**E.G** 
- If we have on _nextjs.config.js_ `['fr-BE']` - all is fine with this locale
- If we have `['fr', 'fr-BE']` - `fr` works fine,  but on `fr-BE` have 404 for all files 

### Additional information

**next.config.js** 
```js
module.exports = {
  i18n: {
    locales: ['en', 'fr', 'fr-BE'],
    defaultLocale: 'en',
  },
};
```
**File structure**
```
/public/hello.txt
```

**Result:**
```
/hello.txt - 200
/en/hello.txt - 200
/fr/hello.txt - 200
/fr-BE/hello.txt - 404
```

### Example
https://00-i18n-files-1hjbrpd9b-nicolas-starofservic.vercel.app/hello.txt
https://00-i18n-files-1hjbrpd9b-nicolas-starofservic.vercel.app/en/hello.txt
https://00-i18n-files-1hjbrpd9b-nicolas-starofservic.vercel.app/fr/hello.txt
https://00-i18n-files-1hjbrpd9b-nicolas-starofservic.vercel.app/fr-BE/hello.txt

**Comment by smaeda-ks** [comment, marked as answer]
@amurashev It looks like the issue has indeed started since that change.
We're now reverting the change, and it may take some time to update our build container version to use the updated Vercel CLI version.

As a temp workaround, you can add an environment variable called "VERCEL_CLI_VERSION" with the value "vercel@28.16.10" on your project. Could you try this and see if this mitigates the problem?

I'll keep you updated with the fix rollout status.

**Reply by amyegan** [reply]
An updated version of the CLI is available now. Please be sure to remove the `VERCEL_CLI_VERSION` environment variable, if you have added one, so the new version will be used on new builds

**Reply by amurashev** [reply]
Thank you @smaeda-ks @amyegan ! 🙏 
Now it works fine 🎉
